tests/unit/dom/select-test.js

```diff
@@ -0,0 +1,146 @@
+import { module, test } from 'qunit';
+import { select, setupContext, teardownContext } from '@ember/test-helpers';
+import { buildInstrumentedElement } from '../../helpers/events';
+import hasEmberVersion from '@ember/test-helpers/has-ember-version';
+import { isIE11 } from '../../helpers/browser-detect';
+
+let clickSteps = ['focus', 'focusin', 'input', 'change'];
+
+if (isIE11) {
+  clickSteps = ['focusin', 'input', 'change', 'focus'];
+}
+
+module('DOM Helper: select', function(hooks) {
+  if (!hasEmberVersion(2, 4)) {
```

Comment:
why are we skipping this for older Ember versions?

Reply:
😅 Im guilty of stealing this from another test, I'll remove this

Reply:
@Turbo87 - Because `setupContext` / `teardownContext` only work when >= Ember 2.4, and these tests (really all of these dom related tests) use `setupContext`.